man/jail.conf.5

```diff
@@ -158,6 +158,9 @@ This sets the age at which bans should be purged from the database.
 
 .SH "JAIL CONFIGURATION FILE(S) (\fIjail.conf\fB)"
 The following options are applicable to any jail. They appear in a section specifying the jail name or in the \fI[DEFAULT]\fR section which defines default values to be used if not specified in the individual section.
+Another way to define default options per jail is to specify those in the section \fI[default.jail-name]\fR.
+The option is looked up in \fI[jail]\fR section, in jail defaults \fI[default.jail]\fR and in global \fI[DEFAULT]\fR section in exactly that order.
```

Comment:
you really like hacking @sebres ;)
this imho even further increases difficulty of comprehension of our configuration files.  At least before it was easy and more or less inline with standard Python's treatment of DEFAULT section specifying default values for sections.  Inclusion of additional files provided remedy to allow to override parameters.   Now we would also get this additional `[default.` construct... blows up jail.conf even further... not sure...
going back to issues which arouse recently which lead to this PR
- per jail port overrides (#1270) - well, a single user missed the point of how values are picked up from DEFAULT.  Given that it is quite a rare report (if not the first), I would say majority of users get the idea (when they do customizations) on how DEFAULT settings work.  So I wouldn't try to fix anything to address it anyhow (besides may be even more documentation? ;))
- backends hardcoded to each section (change in #1225 which lead to #1372) thus making it impossible to change backend in DEFAULT to come in effect for all the sections.  Well, as I have suggested in https://github.com/fail2ban/fail2ban/issues/1372#issuecomment-202951440 , imho the most straightforward approach is just to revert #1225 and ask distro maintainers to define those backends for necessary jails only (since otherwise the backend overload in DEFINE) in per-distribution shipped overload file `jail.d/defaults-distro.conf` .  I already do something like that to enable sshd jail out of the box in Debian packages
```
$> cat jail.d/defaults-debian.conf
[sshd]
enabled = true
```

I am just afraid that adding this default.section feature and by increasing complexity of "defaults handling" we would not only make config file longer and harder to digest and customize but possibly overlook possible gotcha... the simpler we keep it all, more robust it would stay.